package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
+    "start": "docusaurus start --host 0.0.0.0",
```

Comment:
I'm not sure about this. Why do we need this change?

Reply:
Probably not needed, this was done to facilitate me in testing the working of [said change](https://github.com/AdguardTeam/KnowledgeBase/issues/126) on all the devices on my LAN